packages/theme/css/characteristics/actionable-next.css

```diff
@@ -0,0 +1,33 @@
+.salt-theme.salt-theme-next {
```

Comment:
(i know its auto gen) i think i much prefer to keep characteristics in their own files like actionable is :) makes it easier to find and scan, imo